data/data-app/ingress.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: data-app-ingress
+  namespace: prod
+  annotations:
+    kubernetes.io/ingress.class: nginx
+spec:
```

Comment:
Since this is going to be in a cluster with other things you'll need the spec to be more to be more specific, specifically:
```
  tls:
  - hosts:
    - data.gccollab.ca
    secretName: data-app-secret
  rules:
  - host: data.gccollab.ca
    http:
      paths:
      - path: /
        backend:
          serviceName: data-app
          servicePort: 2112
```
This also includes the ssl-related part